proto/expr.proto

```diff
@@ -102,7 +102,7 @@ message ExprNode {
     LPAD = 238;
     RPAD = 239;
     REVERSE = 240;
-    STRPOS = 241;
+    STRPOS = 241 [deprecated = true]; // duplicated with POSITION
```

Comment:
Why `deprecated`, even if duplicated I thought it is permissible, just as an alias?

Reply:
Oh I see, we will just rewrite into `POSITION` in frontend, so we won't use `STRPOS` in protobuf.

Reply:
Consider this as dead / duplicate code cleaning. User facing alias is handled by sqlparser and binder.